clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts

```diff
@@ -8,10 +8,11 @@ import { Briefcase, CollectionRequestParams, IModelScopedOperationParams, OrderB
 export const SPECIAL_VALUES_ME = "me";
 
 /**
- * Valid values for Briefcase owner id filter. Currently the only supported value is `me` which allows to query
- * Briefcases owned by the current user making the request.
+ * Valid owner id values are:
+ * - User id;
+ * - `me` keyword, which specifies to reference the current user. Current user is the user that the access token used for request belongs to.
  */
-export type ValidOwnerIdValue = typeof SPECIAL_VALUES_ME;
+export type ValidOwnerIdValue = string;
```

Comment:
maybe `"me" | string` would make `me` option more discoverable?

Reply:
intelisense just changes it to string, so not sure 
![image](https://github.com/iTwin/imodels-clients/assets/80688723/0dd21ef8-e0d9-4afd-bf3c-ca6ade956042)

Reply:
maybe it's a jsdoc renderer thing, however people do look at the type definitions when trying to figure out how to use client, so I think we should include the special value here.